### PR TITLE
Enter the logic for hash table

### PR DESCRIPTION
**Pull request by silam**
### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by CaedenPH — APPROVED** [review]

**Review by CaedenPH — APPROVED** [review]

**Comment by silam** [comment]
How do I know what I did that I got label: tests are failing

File: data_structures/hashing/double_hash.py [hunk 1]
```diff
@@ -1,4 +1,27 @@
...
+
+Double hashing can be done using : 
+(hash1(key) + i * hash2(key)) % TABLE_SIZE 
+
+Here hash1() and hash2() are hash functions and TABLE_SIZE 
+
+is size of hash table. 
+
+Reference: https://en.wikipedia.org/wiki/Double_hashing#:~:text=Double%20hashing%20is%20a%20computer,data%20structure%20on%20a%20table%20.
+
```
Comment (CaedenPH) [inline data_structures/hashing/double_hash.py:22]:
```suggestion
```

**Review by CaedenPH — APPROVED** [review]
@cclauss This is good

**Comment by cclauss** [comment]
Unfortunately two files changed in one pr.

**Pull request closed without merging**